epregressions/tests/test_runtests.py

```diff
@@ -2071,6 +2071,17 @@ def test_file_does_not_exist(self):
         self.assertEqual(EndErrSummary.STATUS_MISSING, results_for_file.summary_result.simulation_status_case1)
         self.assertEqual(EndErrSummary.STATUS_MISSING, results_for_file.summary_result.simulation_status_case2)
 
+    def test_eio_with_version(self):
```

Comment:
New unit test exercising the EIO diff changes.

Reply:
I do see you changed the version here but this is comparing eio diffs.

Reply:
I'm not sure I understand - the EIO does have a `Version, X.Y` line, and that is what I'm ignoring now.  Could you clarify?

Reply:
This was related to my first comment in the html files, where you corrected me about Version ID. I thought you didn't change it there but did change it here for the eio unit test.

Reply:
Ahh, sorry I misunderstood.  OK, all is well then.  Thanks for watching over this!